modules/schemas/src/main/resources/contexts/nexus/core/resource/v0.1.0.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
`score` property missing
`maxScore` property missing
all the properties related to upload metadata are missing.

Reply:
Also missing:
`results` (array)
`resultId` (uri)
`total` (long)

Reply:
The property you mention should have their own context in nexus/core/search otherwise we will end up with potential collision with instance keys if we include the search vocabulary in the nexus/core/resource context. I suggest to keep the nxv namespace.

Reply:
You can create such nexus/core/search   context.